puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/nightly_packaging.groovy

```diff
@@ -24,7 +24,7 @@ def setup_nightly_build_environment(args) {
 def generate_sourcefiles(args) {
     def sourcefile_paths = []
     def project_name = args.project_name
-    def ruby_version = args.ruby_version ?: env.ruby_version
+    def ruby_version = args.ruby_version ?: '2.5'
```

Comment:
Could you remind me: this is the groovy equivalent of `args.ruby_version || '2.5'`, right?

Reply:
Yep. Called the Elvis operator.